crates/ruff_python_formatter/src/statement/stmt_function_def.rs

```diff
@@ -147,29 +146,31 @@ fn format_function_header(f: &mut PyFormatter, item: &StmtFunctionDef) -> Format
                     [return_annotation.format().with_options(Parentheses::Always)]
                 )?;
             } else {
+                let parenthesize =
+                    if parameters.is_empty() && !comments.ranges().intersects(parameters.range()) {
```

Comment:
What's the reason for using comment.ranges instead of comments.has_comments(paramteters)?

Reply:
Ah I just forgot.

Reply:
Ah I just forgot.

Reply:
Fixed.